guides/doc-Provisioning_Guide/topics/Bare_Metal.adoc

```diff
@@ -291,7 +291,7 @@ Download the boot disk from {ProjectServer} with the `hammer bootdisk host` comm
 
 This creates a boot ISO for your host to use.
 
-Write the ISO to a USB storage device using the *dd* utility or *livecd-tools* if required.
+Write the ISO to a USB storage device using the `dd` utility or `livecd-tools` if required.
```

Comment:
I'm unsure about this in IBM style. I don't have a copy handy. @spetrosi can you verify? I'm pretty sure dd should be bolded in IBM style. Don't know about livecd-tools.

Reply:
IBM Style guide does not know the `` formatting. And we use it commonly for command-line utilities in our docs. We use bold mostly for UI elements. SO I think this is a correct change.

Reply:
This is untrue, Sergei. It is referred to as monospace and that is exactly what is prescribed for commands. This is not the use of a command, but a reference to a utility/tool/command in running text. I'll find the reference myself.

Reply:
Pg 185 onwards of IBM style guide. It gives both examples. In this example, in running text, you use bold font to highlight the command/utility. In an example of the command, you use monospace/``.

Reply:
Actually both bold and monospace! 

"In running text, use a bold monospaced font for a command. If a parameter or option is a variablevalue, use an italic monospaced font; otherwise, use a bold monospaced font for the parameterand a monospaced font for the option." Pp 185 IBM Style Guide

Reply:
I did not have access to this style guide, but I am with Sergei on this one. As it's a command, monospaced fonts are IMHO better suited & bold text is mostly reserved for UI elements.
My only source was [ibm.com styleguidelines](https://www.ibm.com/developerworks/library/styleguidelines/index.html) searching for "command name".

Still, I've reverted my change.

Reply:
Thanks Maximilian. 
I reserve all opinion for when there is a grey area. Here there is no grey area; we have a clear directive from the style guide!

Reply:
Oh seems like we have been violating the IBM style guide for years.
I grepped `grep -r '`*`' guides/doc-*/topics/` and I see many many utilities/tools/commands are enclosed in ``.
I also grepped `grep -r '\**\*' guides/doc-*/topics/` (not the most elegant grep) and there is only a few utilities/tools/commands enclosed in **.
I never used both bold and monospace in Asciidoc, no idea if it would work.

Reply:
Again it only counts in running text, so in the middle of a sentence. Not with the actual command. The monospace and bold font is not that hard to implement, although I am not expecting it as part of this PR.